Model/Api/Client.php

```diff
@@ -86,26 +103,37 @@ private function getApiUrl(Request $request)
     }
 
     /**
-     * @param string $url
-     * @param string $method
-     * @param array $headers
-     * @param string $data
+     * @param TokenInterface $token
+     * @param Request $request
      * @return bool|string
      */
     private function sendHttpRequest(TokenInterface $token, Request $request)
     {
         $isJson = preg_match('/application\/.*json/',$request->getContentType());
+        $content = $isJson ? $this->json->serialize($request->getBody()) : $request->getBody();
         $options = [
             'http' => [
                 'method' => $request->getMethod(),
                 'header' => implode("\r\n", $this->prepareHeaders($token, $request)),
-                'content' => $isJson ? $this->json->serialize($request->getBody()) : $request->getBody(),
+                'content' => $content,
                 'ignore_errors' => true,
                 'timeout' => 120
             ]
         ];
         $context = stream_context_create($options);
 
-        return file_get_contents($this->getApiUrl($request) . $request->getUri(), false, $context);
+        $id = uniqid();
```

Comment:
$requestId = uniqid('', true);
^^ without "more_entropy" set to true it will not work under windows, and it's more probable to get not unique values.

Reply:
Done